frontend/rust-lib/flowy-database2/src/entities/type_option_entities/date_entities.rs

```diff
@@ -94,7 +107,23 @@ impl From<DateTypeOptionPB> for DateTypeOption {
   }
 }
 
-#[derive(Clone, Debug, Copy, EnumIter, ProtoBuf_Enum, Default)]
+#[derive(Clone, Debug, Copy, ProtoBuf_Enum, Default, Serialize, Deserialize)]
+pub enum ReminderOptionPB {
+  #[default]
+  None = 0,
+  AtTimeOfEvent = 1,
+  FiveMinsBefore = 2,
+  TenMinsBefore = 3,
+  FifteenMinsBefore = 4,
+  ThirtyMinsBefore = 5,
+  OneHourBefore = 6,
+  TwoHoursBefore = 7,
+  OneDayBefore = 8,
+  TwoDaysBefore = 9,
+  Custom = 10,
+}
+
+#[derive(Clone, Debug, Copy, ProtoBuf_Enum, Default)]
```

Comment:
I think this should go in a crate that is shared with the document.

Reply:
Why? It is only used and stored in DateTypeOptionPB, and has no other relation whatsoever with Document, Reminder, Database, etc.